spec/features/checkout_spec.rb

```diff
@@ -58,4 +66,44 @@
       expect(response.headers["Location"]).to match(%r(https://www\.sisow\.nl/Sisow/iDeal/Simulator\.aspx\?merchantid=2537407799&txid=\w*&sha1=\w*))
     end
   end
+
+  context "site has has paymentmethod Paypal" do
+    let(:paypal) { Spree::PaymentMethod::SisowBilling::Paypalec.create!(name: 'Paypal') }
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.